solution/ui/regulations/eregs-vite/src/components/SearchInput.vue

```diff
@@ -128,14 +128,16 @@ export default {
 
     methods: {
         submitForm() {
-            this.$emit("execute-search", { query: this.searchInputValue });
+            this.$emit("execute-search", {
+                query: encodeURIComponent(this.searchInputValue),
+            });
         },
         clearForm() {
-            this.searchInputValue = undefined;
+            this.searchInputValue = "";
             this.$emit("clear-form");
         },
         updateSearchValue(value) {
-            this.searchInputValue = value;
+            this.searchInputValue = decodeURIComponent(value);
```

Comment:
always decode v-model value when updated so the value appears as human readable in the input